### PR TITLE
Update .length checks in controllers to use _.size()

### PR DESCRIPTION
**Pull request by benjaminapetersen**
re #1352 
Updating uses of `.length` in controllers to `_.size()` and/or `_.isEmpty()`. 

Opting for consistency by changing all the references (there are few), rather than being particular about only changing those associated w/an API object field.

### CODE REVIEW

File: app/scripts/controllers/buildConfig.js [hunk 1]
```diff
@@ -207,7 +211,7 @@ angular.module('openshiftConsole')
 
         // Sort now to avoid sorting on every digest loop.
         $scope.orderedBuilds = BuildsService.sortBuilds($scope.builds, true);
-        $scope.latestBuild = $scope.orderedBuilds.length ? $scope.orderedBuilds[0] : null;
+        $scope.latestBuild = latestBuild($scope.orderedBuilds);
```
Comment (jwforres) [inline app/scripts/controllers/buildConfig.js]: i think we would be fine just doing `_.first($scope.orderedBuilds)` here and dropping the added method.  we dont need it to be specifically set to null, just some falsey / non-existent value.

**Comment by jwforres** [comment]
We have to sweep all of the javascript, this includes directives/components, filters, services -- and across all three repos.  Do we really want to change *every* reference to length across all of those files?

**Comment by benjaminapetersen** [comment]
Yeah, not really.  Thats a lot of changes.  Starting with some small PRs to see how this goes as I fish around.

File: app/scripts/controllers/buildConfig.js (hunk 1 above)
Comment (benjaminapetersen) [inline app/scripts/controllers/buildConfig.js]: cool, ill update

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
[merge]

**Pull request merged**